CHANGES.md

```diff
@@ -3,6 +3,8 @@ Change Log
 
 ### 1.18 - 2016-02-01
 
+* Deprecated
+    * Deprecated `GroundPrimitive.geometryInstance`. It will be removed in 1.20. Use `GroundPrimitive.geometryInstances` instead.
```

Comment:
Also mention that ground primitive now supports batching for better performance.